sweep/sweeper.go

```diff
@@ -657,7 +657,13 @@ func (s *UtxoSweeper) collector(blockEpochs <-chan *chainntnfs.BlockEpoch) {
 		// we are already trying to sweep this input and if not, set up
 		// a listener to spend and schedule a sweep.
 		case input := <-s.newInputs:
-			s.handleNewInput(input)
+			err := s.handleNewInput(input)
+			if err != nil {
+				log.Errorf("Unable to handle new input: %v",
```

Comment:
`Fatalf`? 

This way we'll actually _shutdown_ vs operating in a partially consistent state.

Reply:
yeah switched